src/main/java/carRacing/controller/RacingCarGameManager.java

```diff
@@ -0,0 +1,48 @@
+package carRacing.controller;
+
+import carRacing.model.MoveStrategy;
+import carRacing.model.RacingCar;
+import carRacing.model.RandomGenerator;
+import carRacing.model.RandomMoveStrategy;
+import carRacing.view.InputView;
+import carRacing.view.ResultView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingCarGameManager {
+
+    InputView in = new InputView();
+
+    public void run() {
+        // 입력
+        int carCount = in.inputNumber(InputView.numberOfCar());
+        int raceCount = in.inputNumber(InputView.numberOfRace());
```

Comment:
```suggestion
        int carCount = InputView.inputNumber(InputView.numberOfCar());
        int raceCount = InputView.inputNumber(InputView.numberOfRace());
```

인스턴스 변수가 없는(즉, 상태가 없는) 객체는 굳이 인스턴스를 생성할 필요가 없다.
굳이 인스턴스를 생성할 필요가 없는 경우 클래스 메서드로 구현해보면 어떨까?